components/org.wso2.carbon.identity.conditional.auth.functions.elk/pom.xml

```diff
@@ -0,0 +1,159 @@
+                        <Private-Package>
+                            org.wso2.carbon.identity.conditional.auth.functions.elk.internal
+                        </Private-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.identity.conditional.auth.functions.elk.internal,
+                            org.wso2.carbon.identity.conditional.auth.functions.elk.*
+                        </Export-Package>
+                        <Import-Package>
+                            javax.servlet.*; version="${imp.pkg.version.javax.servlet}",
+                            org.apache.commons.io,
+                            org.apache.commons.lang,
+                            org.apache.commons.logging,
+                            org.apache.http.entity,
+                            org.apache.http.impl,
+                            org.apache.http.util,
+                            org.apache.http.client,
+                            org.apache.http.client.methods,
+                            org.apache.http.client.config,
+                            org.apache.http.impl.client,
+                            org.apache.http.conn,
+                            org.json.simple,
+                            org.json.simple.parser,
+                            org.apache.http,
+                            org.osgi.service.component,
+                            org.osgi.service.component.annotations,
+                            org.wso2.carbon,
+                            org.wso2.carbon.core,
+                            org.wso2.carbon.core.util,
+                            org.apache.axiom.om.util,
+                            org.wso2.carbon.identity.application.authentication.framework,
+                            org.wso2.carbon.identity.application.authentication.framework.context,
+                            org.wso2.carbon.identity.application.authentication.framework.exception,
+                            org.wso2.carbon.identity.application.authentication.framework.model,
+                            org.wso2.carbon.identity.application.authentication.framework.config.model,
+                            org.wso2.carbon.identity.application.authentication.framework.config.model.graph.js,
+                            org.wso2.carbon.identity.application.authentication.framework.config.model.graph,
+                            org.wso2.carbon.identity.core.util,
+                            org.wso2.carbon.registry.core.service,
+                            org.wso2.carbon.user.core; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.user.core.service; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.identity.conditional.auth.functions.common.utils,
+                            org.wso2.carbon.identity.conditional.auth.functions.common.auth,
```

Comment:
We need to define the relevent version ranges here.

Reply:
Resolved with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/98/commits/1c85a7ec168ab69d824664abc5cb3d4f10a2ef40